### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by AhmedEzzat12** [jaredpalmer/formik#3515] [General]
I'm starting a new project, and I wanted to use formik but I'm afraid if it's not maintained anymore.

**Comment by johnrom** [comment]
It's pretty much dead at the moment. I have a PR open for v3 #3231, but I'm not willing to take over as a maintainer at the moment and the repo owner has been silent for a long time.

That said, I'm still using my v3 PR, so who knows about the future.

**Reply by zlf341** [reply]
let's make the future? :100:

**Comment by zlf341** [comment]
should we create a new project to publish this formik3 and to gain more supporters?

maybe call it, formkit

:)

**Comment by pamapa** [comment]
if you wait too long, you will loose traction against competitor libraries like e.g. https://github.com/react-hook-form/react-hook-form

**Reply by babaliaris** [reply]
> if you wait too long, you will loose traction against competitor libraries like e.g. https://github.com/react-hook-form/react-hook-form

I guess I'm moving to **react-hook-form** as well. It is cleaner and has only 18 pull requests (at the moment) and almost the same stars as Formik, plus everyone on Reddit is happy making this transition.

**Reply by johnrom** [reply]
React hook form is not a competitor library. They are an uncontrolled form library. If you want controlled forms, you cannot use React Hook Form. (this could have changed since I originally looked at it)

https://reactjs.org/docs/uncontrolled-components.html

That said, if uncontrolled forms are your thing, of course you should use it!

**Reply by pamapa** [reply]
I guess that changed: https://react-hook-form.com/api/usecontroller

**Reply by babaliaris** [reply]
I actually returned back to Formik. I spend a day trying to integrate react-hook-form with MUI using the controller render component and still couldn't make it work. For pure HTML input, it was easy but I use UI libraries, so it doens't work for me.

**Reply by johnrom** [reply]
it's cool to know that RHF has controlled forms, I'll have to look at it again. Formik still feels more like using regular React to me without all the proxies and magic, so I'm still considering a fork. I marked my comment above as outdated so as not to confuse future readers.
